packages/react-aria-components/stories/Popover.stories.tsx

```diff
@@ -433,3 +433,28 @@ export const PopoverTriggerWidthExample = () => (
     </Popover>
   </DialogTrigger>
 );
+
+const PopoverRightExampleContent = () => {
+  const [s, ss] = useState(false);
+  useEffect(() => {
+    setTimeout(() => {
+      ss(true);
+    }, 1500);
+  });
+  return s ? (
+    <div>
+      Longer content goes here. Hello, World! This is a popover.
+    </div>
+  ) : (
+    <div>Short content goes here.</div>
+  );
+}
+
+export const PopoverRightExample = () => (
+  <MenuTrigger>
+    <Button style={{ marginLeft: 'auto' }}>Open popover</Button>
+    <Popover placement="bottom end">
+      <PopoverRightExampleContent />
+    </Popover>
+  </MenuTrigger>
+);
```

Comment:
This story is more of a test, to reproduce the issue before the fix and to verify it works correctly after the fix. Is there a better place to do that? If not, what should I name it? I wanted to write a unit test, but I'm not sure how to write it.

Reply:
it's fine to have a storybook example for something like this, it's really hard to reproduce in a unit test

one option could be to use storybook's "play" feature and see if you can use that to automate what should happen